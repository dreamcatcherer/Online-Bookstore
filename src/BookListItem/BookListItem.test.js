import React from "react";
import { render } from "@testing-library/react";
import BooksListItem from "./BookListItem.jsx";

it("renders the BooksListItem component correctly", () => {
  const { container } = render(
    <BooksListItem
      url="https://images-na.ssl-images-amazon.com/images/I/9195S4FTnOL.jpg"
      title="Potty"
    />
  );
  expect(container).toMatchSnapshot();
});
