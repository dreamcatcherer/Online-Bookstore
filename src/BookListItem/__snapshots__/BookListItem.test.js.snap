// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BooksListItem component correctly 1`] = `
<div>
  <div
    class="listItem"
  >
    <div
      class="imageStyle"
    >
      <img
        alt="Potty"
        src="https://images-na.ssl-images-amazon.com/images/I/9195S4FTnOL.jpg"
      />
    </div>
    <p
      class="titleStyle"
    >
      Potty
    </p>
  </div>
</div>
`;
